tokens/tokens-eth.json

```diff
@@ -1,5 +1,37 @@
 [
 {
+"symbol"      : "FLMC",
+"address"     : "0x5976F7dac1525eF3277836043bA474a35E6B4272",
+"decimals"    : "0",
```

Comment:
Was this meant to be zero @boomfly?

Reply:
yes this is correct: https://etherscan.io/tokens?q=0x5976F7dac1525eF3277836043bA474a35E6B4272

but should not be a string - but "decimals":0

Reply:
@ligi Thanks! And why should it be a number? Seems like all other decimals are listed as strings.

Reply:
because it is a number right ;-)
Just because every1 else screws up - you do not need to copy this ..
Corrected them all in - github.com/ethereum-lists

Reply:
@ligi thats true. Just wanted to make sure it doesn't break anything when it's ported over to the list in https://github.com/kvhnuke/etherwallet